behat/tests/features-v2/reporting/sections/documents/documents.feature

```diff
@@ -0,0 +1,74 @@
+@v2
```

Comment:
I have been adding an additional tag here for the individual feature in addition to the v2 tag for ease of  manual running. Totally not necessary but either I should remove the tags or you add them. Up to you :)

Reply:
Ah yeah thats a fair point - I'll update 👍